Description: fixed a few errors

--- pymecavideo-6.1a.orig/src/oooexport.py
+++ pymecavideo-6.1a/src/oooexport.py
@@ -117,13 +117,13 @@ class Calc:
         @param app pointeur vers l'application
         """
         ## fait une ligne de titres
-        self.titles(["t (s)"])
         titles=["t (s)"]
         for i in range(app.nb_de_points):
             x = "X%d (m)" % (1 + i)
             y = "Y%d (m)" % (1 + i)
             titles.append(x)
             titles.append(y)
+        self.titres(titles)
         ## fait les lignes de données
         t=0
         dt = app.deltaT
@@ -139,9 +139,9 @@ class Calc:
         ## accroche la feuille au document tableur
         self.doc.spreadsheet.addElement(self.table)
         ## écrit dans le fichier de sortie
-        self.doc.save(self.outfile.name)
+        self.doc.save(self.outfile)
         self.outfile.close()
-        CalcThread(self.outfile).start()
+        CalcThread(self.outfile.name).start()
         return
         
 
