Description: adjusted the version number (from svn's branch)

Index: pymecavideo-6.1a/setup.py
===================================================================
--- pymecavideo-6.1a.orig/setup.py
+++ pymecavideo-6.1a/setup.py
@@ -1,10 +1,12 @@
 # -*- coding: utf-8 -*-
 ### setup.py ###
 
+import src.version
+
 from distutils.core import setup
 # import py2exe
 setup(name='pymecavideo',
-      version='6.0',
+      version="%s" % src.version.Version,
       description=u"pymecavideo permet de tracer des trajectoires issues de videos et d'en exporter les resultats",
       author='Jean-Baptiste BUTET, Georges KHAZNADAR',
       author_email='ashashiwa@gmail.com, georgesk@ofset.org',
